autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -4,18 +4,89 @@
 from django.core.management import base
 from scraper.banner_requests import BannerRequests
 from scraper.models import Course, Instructor, Section, Meeting
+from course import generate_course_id
```

Comment:
```suggestion
from scraper.models.course import generate_course_id
```

This was causing an error, but should always use absolute imports regardless